trk/src/Screens/ClimbCreate/index.js

```diff
@@ -125,139 +187,169 @@ const ClimbInputData = () => {
   }
 
 
+
+  const confirmDelete = () => {
+    Alert.alert(
+      "Delete Climb",
+      "Are you sure you want to delete this climb?",
+      [
+        {
+          text: "No",
+          onPress: () => console.log("Cancel Pressed"),
+          style: "cancel"
+        },
+        {
+          text: "Yes", onPress: () => handleDeleteClimb()
+        }
+      ],
+      { cancelable: false }
+    );
+  };
+
+  async function handleDeleteClimb() {
+    const { updateClimb } = ClimbsApi();
+    updateClimb(climbData.id, { archived: true })
```

Comment:
Also, when the user goes ahead and deletes the climb, the climb gets a new field called archived. Set to true.